starter-code/scripts/article.js

```diff
@@ -21,6 +21,9 @@ Article.prototype.toHtml = function() {
   //       - Select your template from the DOM.
   //       - Now "compile" your template with Handlebars.
   //       - Don't forget to return your template for this article.
+  var source = $('#articles-template').html();
+  var templateRender = Handlebars.compile(source);
+  return templateRender(this);
```

Comment:
Very nice! Remember that the`Handlebars.compile` method will return a new, custom function based on the html (with expressions) that we pass in.

That new function requires a context object, in this case, our `this` is the context object, referring to the article that invoked our `toHtml()` method.